js/screens/ChildInfo/ChildInfo.js

```diff
@@ -25,70 +31,73 @@ const ChildInfo = ({
-                      childId: child.id,
-                      childName: values.childName,
-                    },
-                  });
+                    await updateChild({
+                      variables: {
+                        childId: child.id,
+                        childName: values.childName,
+                      },
+                    });
 
-                  resetForm();
-                }
-              : () => showEditInfo()
-          }
-          titleStyle={{
-            paddingHorizontal: 2,
-            textAlignVertical: 'center',
-          }}
-        />
-      </View>
-      <ProfileCard
-        avatar={child.avatar.imageurl}
-        action={<ActionButton pressed={() => {}} />}
-      />
-      <View style={styles.formContainer}>
-        {editChildInfo ? (
-          <ChildInfoForm
-            updateFormValues={updateFormValues}
-            child={child}
-            setFormValues={setFormValues}
+                    resetForm();
+                  }
+                : () => showEditInfo()
+            }
+            titleStyle={{
+              paddingHorizontal: 2,
+              textAlignVertical: 'center',
+            }}
           />
-        ) : (
-          <View>
-            <Text style={styles.title}>Parent Name</Text>
-            <Text style={styles.info}>
-              {child.parent.firstname} {child.parent.lastname}
-            </Text>
-            <Text style={styles.title}>Child Name</Text>
-            <Text style={styles.info}>{child.name}</Text>
-            <Text style={styles.title}>Parent Email</Text>
-            <Text style={styles.info}>{child.parent.email}</Text>
-            <Text style={styles.title}>Parent Phone</Text>
-            <Text style={styles.info}>{child.parent.phone}</Text>
-          </View>
-        )}
-      </View>
-    </ScrollView>
+        </View>
+        <ProfileCard
+          avatar={child.avatar.imageurl}
+          action={<ActionButton pressed={() => {}} />}
+        />
+        <View style={styles.formContainer}>
+          {editChildInfo ? (
+            <ChildInfoForm
+              updateFormValues={updateFormValues}
+              child={child}
+              setFormValues={setFormValues}
+            />
+          ) : (
+            <View>
+              <Text style={styles.title}>Parent Name</Text>
+              <Text style={styles.info}>
+                {child.parent.firstname} {child.parent.lastname}
+              </Text>
+              <Text style={styles.title}>Child Name</Text>
+              <Text style={styles.info}>{child.name}</Text>
+              <Text style={styles.title}>Parent Email</Text>
+              <Text style={styles.info}>{child.parent.email}</Text>
+              <Text style={styles.title}>Parent Phone</Text>
+              <Text style={styles.info}>{child.parent.phone}</Text>
+            </View>
+          )}
+        </View>
+      </ScrollView>
+      <View style={{ height: 150 }} />
```

Comment:
Style props should not be added to closing JSX tags. Remove this if it is not being used.

Reply:
This View is actually its own container and the height that is used here is important for the keyboard not to cover the whole textInputs

Reply:
OK thanks for the clarification. This has been merged.